server/routes/charts.js

```diff
@@ -71,9 +71,21 @@ router
   .route('/charts/:chart_id/edit')
   .get(ensureAuthenticated, async (req, res) => {
     try {
+      const { chart_id } = req.params
+      const { dataDb } = req.app.locals
+      const chart = await dataDb.collection(collections.charts).findOne({
+        _id: ObjectID(chart_id),
+        owner: req.user,
+      })
+
+      if (!chart)
+        return res
+          .status(403)
+          .json({ message: 'Not authorized. Please contact the admin' })
```

Comment:
can we redirect to the charts list page here?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/135/commits/0f1c63d004fcc0ce722558e1b2dab472de9388c3